drracket/drracket/private/unit.rkt

```diff
@@ -4711,7 +4711,17 @@
                (if (is-a? f drracket:unit:frame<%>)
                    1
                    0))))
-        
+
+      (define/override (find-editor predicate)
+        (cond
+          [(findf predicate (map (lambda (tab) (send tab get-defs))
+                                 tabs))
+           => (lambda (editor)
+                editor)]
+          [(predicate interactions-text)
+           interactions-text]
+          [else #f]))
+                      
```

Comment:
This look looks like it is searching in every definitions text but only one interactions text. Is that intended?